tests/hardware_tests/configs/waypoints.yaml

```diff
@@ -59,3 +50,16 @@ nav_targets:
   - 3.5000000000000000
   - 0.0000000000000000
   - 0.0000000000000000
+
+  test_place_f:
```

Comment:
Why there are two copies of this?

Reply:
One set is "place_target" in (end_effector_x,end_effector_y.end_effector_z) and
other set is "nav_target" in (base_x, base_y, base_yaw)